SeriesfeedSeasonSeparator.user.js

```diff
@@ -28,7 +28,9 @@ $(function() {
 			var season = /S(.*)E/.exec(tag)[1];
 			
 			if (current !== season) {
-				row.css('border-top', '3px solid #DDD');
+				if (!row.is(':first-child')) {
```

Comment:
This wasn't a bug, it's a feature :P 
It's looks odd to me if the top level doesn't have a separator... But maybe that's just how I got used to it. Will be removing this line, though.

Reply:
It looked a bit weird to me that you'd have an extra thick line after you have the header already distinguishing, but yeah, I don't mind 👍